src/vs/workbench/parts/terminal/test/electron-browser/terminalLinkHandler.test.ts

```diff
@@ -41,6 +65,7 @@ suite('Workbench - TerminalLinkHandler', () => {
 			testLink('c:/a/long/path');
 			testLink('c:\\a\\long\\path');
 			testLink('c:\\mixed/slash\\path');
+			testLink('a/relative/path');
```

Comment:
Actually now that I think about it that's probably not a good idea until we know more about the current working directory within the terminal. It could get confusing otherwise. Right now wealways open files relative to VS Code's folder, not the terminal's.